### PR TITLE
kinder: add workflow for testing super-admin.conf

### PR DESCRIPTION
**Pull request by neolit123**
checks if the right files are generated, they have the correct certificates, that commands work without errors, etc. let me know if you think something else can be added.

tested locally with `kinder test workflow...`

xref https://github.com/kubernetes/kubeadm/issues/2414

test-infra PR:
https://github.com/kubernetes/test-infra/pull/31161

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/lgtm
/approve
/hold 
feel free to cancel hold

File: kinder/ci/tools/update-workflows/templates/workflows/super-admin-tasks.yaml [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+      ${CMD} rm -f /etc/kubernetes/super-admin.conf
+      ${CMD} sudo kubeadm certs renew super-admin.conf || exit 1
+      ${CMD} kubeadm certs check-expiration || exit 1
+
+      # Cleanup
+      ${CMD} rm -f /etc/kubernetes/pki/ca.*
+      ${CMD} rm -f /etc/kubernetes/*.conf
+
+      # Ensure exit status of 0
+      exit 0
```
[the remark this answers is not in the document]
Comment (chendave) [reply kinder/ci/tools/update-workflows/templates/workflows/super-admin-tasks.yaml:89]: seems good as it reached the end of the script here.

**Review by chendave — COMMENTED** [review]
/lgtm

I cannot see any problem from this new workflow.

**Comment by neolit123** [comment]
/hold cancel
let's run this in CI and see how it goes, i will send fixes if needed.
thanks

File: kinder/ci/tools/update-workflows/templates/workflows/super-admin-tasks.yaml [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+  timeout: 5m
+- name: pre-init
+  description: |
+    Run commands before kubeadm init is called on a primary CP node
+  cmd: /bin/bash
+  args:
+    - -c
+    - |
+      set -x
+      CMD=docker exec {{ .vars.clusterName }}-control-plane-1
```
Comment (neolit123) [inline kinder/ci/tools/update-workflows/templates/workflows/super-admin-tasks.yaml:59]: i forgot to add quotes here.
it causes these CMD commands to fail, must have happened when i synced the template and the real workflow in ci/workflows.

https://[storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-kinder-super-admin-latest/1719602416766160896/build-log.txt](https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-kinder-super-admin-latest/1719602416766160896/build-log.txt)

https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-super-admin-latest

will send a PR and self LGTM.

**Pull request merged**